perfmetrics/scripts/run_load_test_and_fetch_metrics.sh

```diff
@@ -2,6 +2,8 @@
 set -e
 echo Running fio test..
 fio job_files/seq_rand_read_write.fio --lat_percentiles 1 --output-format=json --output='output.json'
+echo Logging fio results
+python3 fio_logger.py gcs/Fio-logs/ output.json
```

Comment:
the logic inside fio_logger.py has nothing to do with fio. Its about copying a file and deleting older files. 
I would suggest move the copy step here and handle only keeping last 10 day files there.
Rename the file to metrics_utils.py under utils directory.

Reply:
Done